index.js

```diff
@@ -11,28 +11,171 @@
+    const board = document.getElementById(`board_player${id}`); 
+
+    //Place boats randomly
+    while(player.ships < 4) {
+        const x = Math.floor(Math.random() * 4);
+        const y = Math.floor(Math.random() * 4);
+
+        if (player.board[x][y] == 0) {
+            player.board[x][y] = 1;
+            player.ships += 1;
+        }
+    }
+
+    //const lives = document.querySelectorAll(`#ship_player${id}`).innerHTML = player.lives; // Select the 'Lives' html element and display the number of lives on the page
+    //lives.innerHTML = player.lives;
+
+    for (var x = 0; x < 4; x++) { // Creates the board
+
+        const li = document.createElement('li'); //creating childs for the list (board)
+
+        for (var y = 0; y < 4; y++) {
+            const cell = document.createElement('div');
+            cell.className = 'square'; // adding css properties to make it look like a square
+            cell.textContent = `${x},${y}`//: ${player.board[x][y]}`; // saves the coordinates as a string value 'x,y' and if you uncomment the last piece of code it show if there is a ship inside the cell (a number 1)
+            cell.value = player.board[x][y]; // state of the cell
+            cell.x = x;
+            cell.y = y;
+
+            //This function adds the click event to each cell
+            cell.addEventListener('click', (e) => {
+              if (!player.playing){
+                  return;
+              }
+
+              let cell = e.target; // get element clicked 
+              console.log( cell.textContent) // display the coordinates in the console
+              //cell.style.visibility = 'hidden';
+              cell.style.background ="pink"; // Change the color of the cell when it's clicked
+              
+              if (e.target.value == 1) { // if the cell that is clicked contains a ship (a number 1) 
+                  player.ships -= 1; // The number of ships decreases
+                  player.lives -= 1; // The number of lives decreases
+                  const lives = document.getElementById(`ships_player${id}`); 
+                  lives.innerHTML = player.lives; // Display the number of lives on the page  
+               } 
+
+                player.playing = false;  
+                opponent.playing = true;
+               
+                const turn = document.getElementById('turn_player');
+                turn.innerHTML = opponent.name; // Display who's turn is
+
+                // Display who is the winner based on the number of lives of the opponent
+                if (opponent.lives === 0) { 
+                  turn.innerHTML = `Congratulations ${player.name} ,you WON!!!`;
+                } else {
+                  return;
+                }
+
+                if (player.lives === 0) {
+                  turn.innerHTML = `Congratulations ${opponent.name} ,you WON!!!`;
+                } else {
+                  return;
+                }
```

Comment:
if statements doing the same thing. We can create a function like `checkPlayersLives` that we can call any time.